### DISCUSSION TITLE
How to get all of my old torrents to seed again?

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2467] [Q&A] [answered]
Hello! Sorry if this is a simple question, but when I upgraded to the latest release, Transmission and OpenVPN booted up fine, but all of my torrents were gone from the webUI. I still have all of the actual downloaded files, all of the torrent files, and all of the resume files.

How can I get Transmission to see the resume files and get Transmission back to seeding all my torrents like normal?

Also, since this may be related, \transmission-home is still in my data folder along with my downloads folder, incomplete folder, etc. Could this be the root of the problem? So am I supposed to make a new folder called \config and put \transmission-home in there, and then make another folder called \data and put everything else in there? (and of course change the mapped volumes).

Thanks in advance.

**Comment by ilike2burnthing** [comment, marked as answer]
Add a `/config` volume and move the `transmission-home` folder to it - https://github.com/haugene/docker-transmission-openvpn#docker-run

https://github.com/haugene/docker-transmission-openvpn/discussions/1937#discussioncomment-4263241

**Comment by haugene** [comment]
I see you marked this as answered and that probably means you got it working again by moving to the /config folder. This was supposed to have a fallback though and you probably experienced the same bug as reported here https://github.com/haugene/docker-transmission-openvpn/issues/2409. For anyone reading this I'll just include here that I think we have a fix currently on the dev branch (and available on the `dev` tag of the image) ref my [last comment](https://github.com/haugene/docker-transmission-openvpn/issues/2409#issuecomment-1360452462) in the issue
